ZupIT/ritchie-formulas
# Fix java8 template and add template java11

# Pull Request

## What I did
- Fixed java 8 template to build dependencies with the formula.
- Added variation of the template with configured java11.

## How I did it

- To solve the dependency problem, the maven-assembly-plugin plugin was added to build the jar with the formula dependencies.
- To facilitate the creation of a formula with java11, a variation of the template was added, with specific docker and pom.xml configurations.

## How to verify it

Create a formula using dependencies with the templates java8 or java11 and check that in the final .jar file that all dependencies have been embedded.

## Description for the changelog

Fix dependencies on tempalte java and support to java11
